src/Compilers/CSharp/Portable/Generated/BoundNodes.xml.Generated.cs

```diff
@@ -9005,7 +8996,11 @@ internal abstract partial class BoundTreeWalker: BoundTreeVisitor
             this.VisitList(node.SwitchArms);
             return null;
         }
-        public override BoundNode? VisitDecisionDag(BoundDecisionDag node) => null;
+        public override BoundNode? VisitDecisionDag(BoundDecisionDag node)
+        {
+            this.Visit(node.RootNode);
+            return null;
+        }
```

Comment:
Are these code changes the result of removing the space? If so, we should throw an exception in the `BoundNode` generator if the name fields contain spaces.

Reply:
Agreed. However, this change is harmless as we don't use the visitor mechanism for decision dags or its nodes.

Reply:
> Are these code changes the result of removing the space?

Yes

Reply:
[https://github.com/dotnet/roslyn/issues/43986](https://github.com/dotnet/roslyn/issues/43986)

---
In reply to: [420325700](https://github.com/dotnet/roslyn/pull/43982#discussion_r420325700) [](ancestors = 420325700)